docs/csharp/how-to/index.md

```diff
@@ -63,7 +63,7 @@ You may need to convert an object to a different type.
 - [Convert a string to a `DateTime`](../../standard/base-types/parsing-datetime.md).
 - [Convert a byte array to an int](../programming-guide/types/how-to-convert-a-byte-array-to-an-int.md).
 - [Convert a string to a number](../programming-guide/types/how-to-convert-a-string-to-a-number.md).
-- [Use `as` and `is` to safely cast to a different type](../programming-guide/types/how-to-safely-cast-by-using-as-and-is-operators.md).
+- [Use pattern matching `as` and `is` to safely cast to a different type](../how-to/safely-cast-using-pattern-matching-is-and-as-operators.md).
```

Comment:
If the work "operators" is not used, then punctuation bits would make the link text easier to read:
"Use pattern matching, `as`, and `is` to safely cast to a different type"